ql/src/test/org/apache/hadoop/hive/ql/TxnCommandsBaseForTests.java

```diff
@@ -162,9 +163,23 @@ protected String getWarehouseDir() {
    * takes raw data and turns it into a string as if from Driver.getResults()
    * sorts rows in dictionary order
    */
-  List<String> stringifyValues(int[][] rowsIn) {
-    return TestTxnCommands2.stringifyValues(rowsIn);
+  public static List<String> stringifyValues(int[][] rowsIn) {
+    assert rowsIn.length > 0;
+    int[][] rows = rowsIn.clone();
+    Arrays.sort(rows, new TestTxnCommands2.RowComp());
```

Comment:
move RowComp from TestTxnCommands2 to TxnCommandsBaseForTests

Reply:
fixed